io.github.mpobaschnig.Vaults.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "io.github.mpobaschnig.Vaults",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "vaults",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--filesystem=host"
```

Comment:
do you really need "host" access?

Reply:
I need this so user can access e.g. USB drives under /run or other locations like /mnt where they have access to. I'm not sure if I should set "home" access instead and additionally enable specific directories, but I don't know which directories users might come up with.